### PR TITLE
sesman: add hostname to vnc password file

### PR DESCRIPTION
**Pull request by jsorg71**

### CODE REVIEW

File: sesman/env.c [hunk 1]
```diff
@@ -184,9 +194,8 @@ env_set_user(const char *username, char **passwd_file, int display,
                                         "password file %s", *passwd_file);
                             g_file_delete(*passwd_file);
                         }
-
-                        g_sprintf(*passwd_file, "%s/.vnc/sesman_%s_passwd:%d",
-                                  pw_dir, username, display);
+                        g_sprintf(*passwd_file, "%s/.vnc/sesman_%s_passwd:%d:%s",
+                                  pw_dir, username, display, hostname);
```
Comment (metalefty) [inline sesman/env.c]: bikeshed discussion 😜

It doesn't affect the fix at all but HOSTNAME DISPLAY order looks more natural for me. 
For example, `DISPLAY` environment variable sometimes includes hostname.

Comment (metalefty) [reply sesman/env.c]: @jsorg71  what about like this? just a cosmetic point.

```
g_snprintf("%s/.vnc/sesman_passwd-%s@%s:%d", pw_dir, username, hostname, display);
```

I'd like to ship this to v0.9.6 release.

Comment (jsorg71) [reply sesman/env.c]: sure, looks safe

**Comment by metalefty** [comment]
Merge and I'll change the filename later.

**Pull request merged**